packages/@webex/plugin-cc/src/WebRTCCalling.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  createClient,
```

Comment:
why do we need this file here ? its not part of the scope for now , i can confirm and get back